### ISSUE TITLE
Disable individual segments

### ISSUE THREAD

**Issue by dmacgeo** [alexrainman/SegmentedControl#66]
Hi,

Is it possible to disable one or more individual segments of a segmented control? I tried setting IsEnabled to false  for  some of the control's children but it did not work. I tried this for Android using Xamarin.Forms.

Thanks for the great work on this control!

**Comment by zanesc** [comment]
I am looking to do the exact same thing on the same platform and have tried exactly what you have with the same results.

Did you ever find a way?

**Comment by zanesc** [comment]
My workaround for now will be displaying an alert if the user selects the disabled option then setting the segment control back to the previously selected segment by keeping a track of the previous segment.

**Comment by dmacgeo** [comment]
No, I did not find a way. The design of my project changed a lot and segmented controls are no longer used.
